tests/apps/pubsub-subscriber/app.go

```diff
@@ -57,12 +62,20 @@ func indexHandler(w http.ResponseWriter, r *http.Request) {
 func configureSubscribeHandler(w http.ResponseWriter, r *http.Request) {
 	log.Printf("configureSubscribeHandler called\n")
 
-	var t topicsList
-	t.Topic = append(t.Topic, pubsubA, pubsubB)
-	log.Printf("configureSubscribeHandler subscribing to:%v\n", t.Topic)
+	t := []subscription{
+		subscription{
+			Topic: pubsubA,
+			Route: pubsubA,
+		},
+		subscription{
+			Topic: pubsubB,
+			Route: pubsubB,
+		},
```

Comment:
These test scenarios are same as before, so it is not testing when topic != route. 
Can we add one more test when topic != route ?

Reply:
We didn't test prior for topic != route. (the user subscribing for "/A" but listening to "/B").

In that case, a message will not arrive.